### DISCUSSION TITLE
Different behavior with Box background color between Windows & Linux ?

### DISCUSSION THREAD

**Discussion by ikus060** [beeware/toga#2424] [Q&A]
Hello, On Linux, the background color or Box seams to propagate to children widget. But not on Windows.

Here Linux:
![image](https://github.com/beeware/toga/assets/921251/e43bb5ac-0bf7-4b6f-9d71-28fd51c60cf0)

Here Windows:
![image](https://github.com/beeware/toga/assets/921251/814aec9f-72f1-4ed6-80c5-fdcb2f37c6da)

Is this kind of behavior expected ?

**Comment by freakboy3742** [comment]
That's definitely a bug - at the very least, the behavior is inconsistent. It's possible this is a manifestation of #767; if it's not, it's at least in the same family of problems.

Could I ask you to open an issue with a reproduction case (such as the code for the sample project you have here)?

**Reply by ikus060** [reply]
I've open ticker #2425

**Comment by ikus060** [comment]
MacOS Also seams to be correct:
![image](https://github.com/beeware/toga/assets/921251/86f4ae36-fc11-4b54-bf13-e3a0eb9cfb26)

Android too:
![image](https://github.com/beeware/toga/assets/921251/0f8792eb-146d-46a0-b369-e9364f391d1b)
